app/models/form/sales/pages/buyer_company.rb

```diff
@@ -0,0 +1,15 @@
+class Form::Sales::Pages::BuyerCompany < ::Form::Page
```

Comment:
Please could you add specs for the page and question objects here? You can use https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/885/files#diff-1732d36de4a2d8798f802b2beb2d85fc7547dc6a3640caa318d2304a16ee1c1d as an example.

Reply:
added - I left out the :in-progress sales log in the sales_log.rb factory as that's shared behaviour so can just be picked up from Kat's PR